webapp/tests/karma/ts/services/android-api.service.spec.ts

```diff
@@ -1,35 +1,36 @@
-describe('AndroidApi service', () => {
+import { TestBed } from '@angular/core/testing';
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+import { AndroidApiService } from '@mm-services/android-api.service';
+import { SessionService } from '@mm-services/session.service';
+import { GeolocationService } from '@mm-services/geolocation.service';
+import { FeedbackService } from '@mm-services/feedback.service';
 
-  'use strict';
+describe('AndroidApi service', () => {
```

Comment:
Is it possible or worth adding unit test for any of these:
-  back
- logout
- mrdtResponse
- mrdtTimeTakenResponse
- simprintsResponse
- smsStatusUpdate
- locationPermissionRequestResolved

Reply:
I added a bunch of tests, for each of these. The `back` is super tricky to test because of all the jquery, so I just added tests to cover router navigation.